pkgs/servers/mqtt/mosquitto/default.nix

```diff
@@ -70,12 +66,18 @@ stdenv.mkDerivation rec {
     libuv
     libwebsockets'
     openssl
+    uthash
   ] ++ lib.optional withSystemd systemd;
 
   cmakeFlags = [
-    "-DWITH_THREADING=ON"
-    "-DWITH_WEBSOCKETS=ON"
-  ] ++ lib.optional withSystemd "-DWITH_SYSTEMD=ON";
+    (lib.cmakeBool "WITH_BUNDLED_DEPS" false)
+    (lib.cmakeBool "WITH_WEBSOCKETS" true)
+  ] ++ lib.optional withSystemd (lib.cmakeBool "WITH_SYSTEMD" true);
+
+  postFixup = ''
+    echo "Fixing pkg-config files"
+    sed -i "s|^prefix=.*|prefix=$lib|g" $dev/lib/pkgconfig/*.pc
```

Comment:
It would be super awesome if we could fix the cmake file that generates this and upstream it.